src/video/cocoa/wnd_quartz.mm

```diff
@@ -346,6 +346,10 @@ - (void)drawRect:(NSRect)invalidRect
 		[ this->window makeKeyAndOrderFront:nil ];
 	}
 
+	// Set the window colorspace
```

Comment:
Comments on their own line should use /* */ form, but this comment is useless as is. Comments should say *why*, not what